codespell_lib/data/dictionary.txt

```diff
@@ -26715,9 +26722,11 @@ timeot->timeout
 timeoutted->timed out
 timere->timer
 timeschedule->time schedule
+timespanp->timespan
+timespanps->timespans
 timestan->timespan
-timestanp->timespanp
-timestanps->timespanps
+timestanp->timespan
+timestanps->timespans
```

Comment:
Seems like
```
timestanp->timestamp
timestanps->timestamps
```
might be better here?

Reply:
you are right, i just made it consistent with the corrections in the lines before.